### PR TITLE
Derrick

### PR DESCRIPTION
**Pull request by danubis**
edited line QUEST_LV_0200_20151001_009443 and QUEST_LV_0200_20151001_009448.

### CODE REVIEW

**Comment by danubis** [comment]
updated QUEST_LV_0200.tsv

**Comment by Sunaries** [comment]
Good to merge. 👍

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -9436,12 +9436,12 @@ QUEST_LV_0200_20151001_009439	I will be okay if I take little more rest.{nl}I am
 QUEST_LV_0200_20151001_009440	$Stogas Commanding Officer Ades
 QUEST_LV_0200_20151001_009441	You've brought the letter from Mesafasla?{nl}Then what happened to Roches?{nl}
 QUEST_LV_0200_20151001_009442	Ah... It's fortunate to hear that you are okay.
-QUEST_LV_0200_20151001_009443	You are a foreigner for a long time {nl}The supplies has been stopping from the kingdom, we don't know what's going on..{nl}
+QUEST_LV_0200_20151001_009443	$You have been a foreigner for a long time {nl}There has been no supply from the kingdom, we don't know what's going on..{nl}
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:9439]: Period at the end of the first sentence.  Also might want to look at the original on this one, "You have been a foreigner for a long time" is kind of strange?

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -9436,12 +9436,12 @@ QUEST_LV_0200_20151001_009439	I will be okay if I take little more rest.{nl}I am
...
 QUEST_LV_0200_20151001_009444	Even I'm trying to stand by doing self-sufficiency, but It's hard to alive only myself.{nl}If you are okay, please solve the lack of foods.
 QUEST_LV_0200_20151001_009445	Please get the meat from Beadbird around here.{nl}Horas told me It has a good taste.{nl}
 QUEST_LV_0200_20151001_009446	We will starve, if the Beadbird.{nl}Here is too cold, so only Potatos are growing.
 QUEST_LV_0200_20151001_009447	Thank you.{nl}Firsly, we have to complete it.
-QUEST_LV_0200_20151001_009448	We are not only lacking food.{nl}The weapons also needed to be changed because of no supplies from camp.
+QUEST_LV_0200_20151001_009448	$We are not only lacking food.{nl}The weapons also needed to be improved because of no supplies from camp.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:9444]: Hmm, second sentence needs rewording.
"The weapons situation could also improve, as we've had no supplies from camp."

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -9648,19 +9648,19 @@ QUEST_LV_0200_20151001_009651	The supply officer, Ronda is worried about the foo
...
-QUEST_LV_0200_20151001_009661	Ronda wants to make more caltrops by collecting the fragments of the bars. Collect the fragments of the bars from the bars at Nuvask Cliffs.
+QUEST_LV_0200_20151001_009661	$Ronda wants to make more caltrops by collecting the fragments of the bars. Collect the fragments from the bars at Nuvask Cliffs.
 QUEST_LV_0200_20151001_009662	Hand them over to Ronda
 QUEST_LV_0200_20151001_009663	You've collected the fragments of the bars that have decent qualities. Hand them over to Ronda.
 QUEST_LV_0200_20151001_009664	Talk with Ades
-QUEST_LV_0200_20151001_009665	It seems that Ades is looking for someone who could help him Talk with Ades.
+QUEST_LV_0200_20151001_009665	$It seems that Ades is looking for someone who could help him. Talk with Ades.
 QUEST_LV_0200_20151001_009666	Collect the body fluid of Worg
-QUEST_LV_0200_20151001_009667	Ades wants to cure the magician who is sick due to fever. Collect the body fluid of Worgs that are proven to be effective for reducing fever.
+QUEST_LV_0200_20151001_009667	$Ades wants to cure the magician who has fever. Collect the body fluid of Worgs that are proven to be effective for reducing fever.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:9663]: has a fever

**Comment by Sunaries** [comment]
Is this going to be updated? @danubis

**Comment by danubis** [comment]
Yeah, I have been busy this week.  I will fix them tonight.

发自我的 iPhone

**Comment by Sunaries** [comment]
@danubis I've added your edits with some minor tweaks in my PR. https://github.com/Treeofsavior/EnglishTranslation/pull/1294

This can be closed.

**Comment by Sunaries** [comment]
@imcgames Can you please close this? I've already added the edits of this user in an already merged pull request. :smiley:

**Comment by imcgames** [comment]
Closed as request by @Sunaries

**Pull request closed without merging**